### DISCUSSION TITLE
Playback RAW recording file

### DISCUSSION THREAD

**Discussion by simesa** [KejPi/AbracaDABra#55] [Q&A]
Having recorded a RAW file (on MacOS) - I'm struggling to work out how to make use of it!

I presume it needs a dedicated decoder software but not sure where to start. One app I tried didn't like the file.

Thanks!

**Comment by KejPi** [comment]
AbracaDABra can play tbihe file and as far as I know any DAB SDR with support for RAW files can play them. The file is either uint8 IQ (from RTL SDR devices) or int16 little endian IQ from Airspy or SoapySDR.  What input device have you used to record the IQ stream and what application have you used to replay?
